### PR TITLE
Wrapper around the AG push libs

### PR DESCRIPTION
**Pull request by edewit**
Api:

```csharp
private async void InitApp()
{
      await FHClient.Init();
      FHClient.Push(NotificationReceived);
}

void NotificationReceived(object sender, PushReceivedEvent pushEvent)
{
   Debug.WriteLine(e.Args.message);
}
```

### CODE REVIEW

**Comment by edewit** [comment]
@wei-lee wdyt?

File: FHSDK/FH.cs [hunk 1]
```diff
@@ -319,6 +321,11 @@ public static FHAuthSession GetAuthSession()
 			return headers;
 		}
 
+        public static async void Push(EventHandler<PushReceivedEvent> HandleNotification)
```
Comment (wei-lee) [inline FHSDK/FH.cs]: Might be better call it PushRegister to keep it consistant with ios and android sdks.

**Comment by wei-lee** [comment]
:+1:

**Comment by edewit** [comment]
@wei-lee added demo to the wp8 project

**Comment by wei-lee** [comment]
looks good

**Pull request merged**